pymc/sampling.py

```diff
@@ -2124,17 +2131,33 @@ def init_nuts(
     if init == "auto":
         init = "jitter+adapt_diag"
 
-    _log.info(f"Initializing NUTS using {init}...")
+    if seeds is None:
+        seeds = model.rng_seeder.randint(2 ** 30, dtype=np.int64, size=chains)
+    if not isinstance(seeds, (list, tuple, np.ndarray)):
+        raise ValueError(f"The `seeds` must be array-like. Got {type(seeds)} instead.")
+    if len(seeds) != chains:
+        raise ValueError(
+            f"Number of seeds ({len(seeds)}) does not match the number of chains ({chains})."
+        )
 
-    if random_seed is not None:
-        random_seed = int(np.atleast_1d(random_seed)[0])
+    _log.info(f"Initializing NUTS using {init}...")
 
     cb = [
         pm.callbacks.CheckParametersConvergence(tolerance=1e-2, diff="absolute"),
         pm.callbacks.CheckParametersConvergence(tolerance=1e-2, diff="relative"),
     ]
 
-    # TODO: Consider `initvals` for selecting the starting point.
+    initial_points = get_initial_points(
+        model=model,
+        initvals=initvals,
+        rvs_to_jitter={
+            rv
+            for rv in model.free_RVs
+            if hasattr(rv.tag, "transform")
+            # TODO: Also jitter untransformed RVs as long as they have a real-valued support
+        },
+        seeds=seeds,
+    )
```

Comment:
```suggestion    
    initial_points = get_initial_points(
        model=model,
        initvals=initvals,
        rvs_to_jitter={},
        seeds=seeds,
    )
```

And then re-introduce the jitter again in another PR?